llvm/include/llvm/IR/IntrinsicsAArch64.td

```diff
@@ -3874,11 +3874,47 @@ class SME2_FP8_FDOT_LANE_VG1x4 :
                                llvm_i32_ty],
                           [IntrInaccessibleMemOnly, IntrHasSideEffects, ImmArg<ArgIndex<6>>]>;
 
+class SME2_FP8_FDOT_SINGLE_VG1x2 :
```

Comment:
I have notice that this is also implemented in 
for single: https://github.com/llvm/llvm-project/pull/119568/files
for multi: https://github.com/llvm/llvm-project/pull/119546/files
I think one of you should rebase when merging.